site/content/posts/2023-09-18-multiple-idps-and-identity-transformations.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Pinniped v0.26.0: Multiple identity providers and identity transformations"
+slug: multiple-idps-and-identity-transformations
+date: 2023-09-18
+author: Ryan Richard
+image: https://plus.unsplash.com/premium_photo-1661962912663-49c457dda9ca?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=4144&q=80
+excerpt: "With the release of v0.26.0, Pinniped now supports multiple identity providers and identity transformations"
+tags: ['Ryan Richard', 'release']
+---
+
+![multiple seals](https://plus.unsplash.com/premium_photo-1661962912663-49c457dda9ca?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=4144&q=80)
+*Photo from [Unsplash](https://unsplash.com/photos/fcmxfsAgxqU)*
+
+Pinniped's v0.26.0 release includes several significant new features described below.
+Additionally, the release includes several dependency updates and fixes.
+See the [release notes](https://github.com/vmware-tanzu/pinniped/releases/tag/v0.26.0) for more details.
+
+## Background: Identity provider resources
+
+In Kubernetes, a user identity is a username string and a list of group name strings.
+
+Each OIDCIdentityProvider, LDAPIdentityProvider, or ActiveDirectoryIdentityProvider configures how to use a specific
+protocol (e.g. OIDC or LDAP) to authenticate users and to extract their identities (usernames and group names) for
+Kubernetes from those external identity providers.
+
+Prior to this release, the Pinniped Supervisor would only allow the use of a single OIDCIdentityProvider, LDAPIdentityProvider,
+or ActiveDirectoryIdentityProvider at a time. With this new release, you can now use multiple simultaneous identity
+providers from which to draw user identities.
+
+## New: Multiple FederationDomains
+
```

Comment:
Headings are really powerful for search.  `New: Multiple FederationDomains` is true, but maybe not the bang for the buck we could get.  How about something like:

```
Multiple Identity Providers Configured in Multiple Ways!
```

A little market-y cheese-y perhaps, but might catch a user's attention?  My thinking is to shift from "what" to "why" in the heading, and let the paragraph text elaborate on the "what".